### ISSUE TITLE
Browserslist config is not respected when using eslint-plugin-compat

### ISSUE THREAD

**Issue by jmcantrell** [microsoft/vscode-eslint#1048]
When editing a file, eslint reports that features are incompatible with the current browser settings when it should not.

When running eslint at the command line, no warnings are shown.

In `.browserslistrc`, I have:
```
last 1 chrome version
last 1 firefox version
last 1 safari version
```

![Screen Shot 2020-08-17 at 1 47 13 AM](https://user-images.githubusercontent.com/33200/90365938-42ac6a80-e02c-11ea-8e7e-3358e2d51c37.png)

**Comment by dbaeumer** [comment]
Can you please provide me with a GitHub repository I can clone that demos what you are experiencing.
